pyramid/view.py

```diff
@@ -185,14 +185,21 @@ def my_view(context, request):
     :meth:`pyramid.config.Configurator.add_view`.  If any argument is left
     out, its default will be the equivalent ``add_view`` default.
 
-    An additional keyword argument named ``_depth`` is provided for people who
-    wish to reuse this class from another decorator.  The default value is
-    ``0`` and should be specified relative to the ``view_config`` invocation.
-    It will be passed in to the :term:`venusian` ``attach`` function as the
-    depth of the callstack when Venusian checks if the decorator is being used
-    in a class or module context.  It's not often used, but it can be useful
-    in this circumstance.  See the ``attach`` function in Venusian for more
-    information.
+    Two additional keyword arguments which will be passed to the
+    :term:`venusian` ``attach`` function are ``_depth`` and ``category``.
+
+    ``_depth`` is provided for people who wish to reuse this class from another
+    decorator. The default value is ``0`` and should be specified relative to
+    the ``view_config`` invocation. It will be passed in to the
+    :term:`venusian` ``attach`` function as the depth of the callstack when
+    Venusian checks if the decorator is being used in a class or module
+    context. It's not often used, but it can be useful in this circumstance.
+
+    ``category`` sets the decorator category name. It can be useful in
+    combination with the ``category`` argument to ``scan`` to control which
+    views should be processed.
```

Comment:
This grammar reads odd for me. Should this be instead, "argument *of* ``scan``"?

```
... It can be useful in
combination with the ``category`` argument of ``scan`` to control which
views should be processed.
```